pkg/cmd/server/api/validation/validation.go

```diff
@@ -82,6 +85,19 @@ func ValidateHTTPServingInfo(info api.HTTPServingInfo) fielderrors.ValidationErr
 	return allErrs
 }
 
+func ValidateDisabledFeatures(disabledFeatures []string) fielderrors.ValidationErrorList {
```

Comment:
Ahh.. overlooked this advice previous time. Good idea. I'll make another update.